### PR TITLE
Add: Patrick Collins Yt

### LINKED ISSUE #1760: [Add] Patrick Collins Yt Channel
### Site URL

https://www.youtube.com/@PatrickAlphaC

### Description of the above provided site

This channel offers comprehensive videos for individuals interested in beginning their journey and acquiring expertise in web3 development and blockchain, covering both beginner and advanced levels.

### Category

Youtube

### Sub Category

Web3

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)

### PR DESCRIPTION
**Pull request by Shiva-Sai-ssb**
## Fixes Issue

Closes #1760 

## Changes proposed

I have added Patric Collins' YouTube channel to the project.

## Note to reviewers

I am taking part in Hacktoberfest 2023, kindly include the corresponding label.

### CODE REVIEW

**Review by CBID2 — APPROVED** [review]
Well done @Shiva-Sai-ssb 

**Update**: ignore this comment

**Review by CBID2 — CHANGES_REQUESTED** [review]
Ooh sorry @Shiva-Sai-ssb. The commit from your PR about Codcademy. I recommend doing the git revert command to fix it. You can learn more here: https://www.freecodecamp.org/news/git-revert-how-to-reset-a-file-or-commit/

**Comment by Shiva-Sai-ssb** [comment]
@CBID2 , does this fix?

**Comment by CBID2** [comment]
> @CBID2 , does this fix?

Yup

File: database/youtube/web3-metaverse.json [hunk 1]
```diff
@@ -86,5 +86,13 @@
     "category": "youtube",
     "subcategory": "web3-metaverse",
     "language": "english"
+  },
+  {
+    "name": "Patrick Collins",
+    "description": "This channel offers comprehensive videos for individuals interested in beginning their journey and acquiring expertise in web3 development and blockchain, covering both beginner and advanced levels.",
```
Comment (CBID2) [inline database/youtube/web3-metaverse.json:92]:
```suggestion
    "description": "This channel offers comprehensive videos for individuals interested in acquiring expertise in Web3 development and Blockchain.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
You're on the right track @Shiva-Sai-ssb. I just gave you some tips on improving your chosen source's description.

**Review by CBID2 — DISMISSED** [review]
Well done @Shiva-Sai-ssb! :)

**Comment by Shiva-Sai-ssb** [comment]
Hey @CBID2, I've updated the description as per your suggestion. Will the merged Codecademy pull request be affected by my commit reversal?

**Comment by CBID2** [comment]
> Hey @CBID2, I've updated the description as per your suggestion. Will the merged Codecademy pull request be affected by my commit reversal?

No @Shiva-Sai-ssb. It's already merged :)

**Comment by Shiva-Sai-ssb** [comment]
> > Hey @CBID2, I've updated the description as per your suggestion. Will the merged Codecademy pull request be affected by my commit reversal?
> 
> No @Shiva-Sai-ssb. It's already merged :)

 okay, thank you for your assistance.

File: database/open_source/projects.json [hunk 2]
```diff
@@ -98,7 +98,7 @@
     "subcategory": "projects"
   },
   {
-    "name": "Codecademy Docs",
+    "name": "Codeacademy Docs",
```
Comment (Anmol-Baranwal) [inline database/open_source/projects.json:101]:
```suggestion
    "name": "Codecademy Docs",
```

**Review by Anmol-Baranwal — CHANGES_REQUESTED** [review]
@Shiva-Sai-ssb 

It's `codecademy` not `codeacademy`

image

**Comment by CBID2** [comment]
> @Shiva-Sai-ssb 
> 
> 
> 
> It's `codecademy` not `codeacademy`
> 
> 
> 
> image
> 
> 

@Anmol-Baranwal, that change was made in a different PR

**Comment by Anmol-Baranwal** [comment]
> > @Shiva-Sai-ssb
> > It's `codecademy` not `codeacademy`
> > image
> 
> @Anmol-Baranwal, that change was made in a different PR: #1707

Yeah! @Shiva-Sai-ssb changed that, so she needs to revert to correct the spelling of brand Codecademy.

**Comment by Shiva-Sai-ssb** [comment]
> > > @Shiva-Sai-ssb
> > > It's `codecademy` not `codeacademy`
> > > image
> > 
> > 
> > @Anmol-Baranwal, that change was made in a different PR: #1707
> 
> Yeah! @Shiva-Sai-ssb changed that, so she needs to revert to correct the spelling of brand Codecademy.

What do I do next? Should I rename Codecademy?

**Comment by Anmol-Baranwal** [comment]
> > Yeah! @Shiva-Sai-ssb changed that, so she needs to revert to correct the spelling of brand Codecademy.
> 
> What should I do? Should I rename Codecademy?

You have changed `codecademy` to `codeacademy`.

> what you did.
image

> check ur deployed link
image

Leave it, I have done it.

**Review by Anmol-Baranwal — APPROVED** [review]
🚀 Looks good to me.

**Comment by Anmol-Baranwal** [comment]
@CBID2 
Good to go now.

**Review by CBID2 — APPROVED** [review]
Well done

**Comment by CBID2** [comment]
Thanks for the contribution @Shiva-Sai-ssb!😊 It was wonderful working with you! 😄 If you want to stay updated on LinksHub, join our [Discord community](https://discord.com/invite/NvK67YnJX5)! :) We'd love to have you! :)

**Pull request merged**